### PR TITLE
removed old stuff from the sign up page

### PR DESCRIPTION
**Pull request by sfp-kemp**

### CODE REVIEW

File: app/views/layouts/form_only.html.haml [hunk 1]
```diff
@@ -20,10 +20,11 @@
 
   %body#form-only{ class: I18n.locale }
     %header#top
-      %a{ href: root_path }
-        %img(src="/assets/header/logo400-blue@2x.png" width="400" height="49" alt="Energy Transition Model")
+      .container
```
Comment (antw) [inline app/views/layouts/form_only.html.haml:23]: Personally, I think I preferred the logo center-aligned. It doesn't look quite on the left. The gears in the 400px logo are slightly too wide, which causes "Energy Transition Model" to be misaligned with the content inside the "Create your account" container.

File: app/views/users/new.html.haml [hunk 2]
```diff
@@ -1 +1,21 @@
...
+
+      .row
+        = f.input :password, required: true
+        = f.input :password_confirmation, required: true
+
+    .section.highlight
+      .row
+        = f.input :company_school
+
+    = f.input :allow_news
```
Comment (antw) [inline app/views/users/new.html.haml:19]: `allow_news` checkbox should only be shown in environments where it is supported (i.e. not staging).

```suggestion
    - if MailchimpService.configured?
      = f.input :allow_news
```

File: config/locales/en_users.yml [hunk 3]
```diff
@@ -44,19 +44,6 @@ en:
...
-      unsubscribe: Unsubscribe
-      subscribed: |
-        You are subscribed to our newsletter. We send roughly four
-        e-mails a year with information about improvements and changes to the
-        model.
-      not_subscribed: |
-        We occasionally publish a newsletter with information about
-        improvements and changes to the ETM. We send roughly four messages a
-        year and you can unsubscribe at any time.
-
```
Comment (antw) [inline config/locales/en_users.yml:59]: These texts are used on the "My account" page.

**Comment by sfp-kemp** [comment]
Won't merge this for now.

**Pull request closed without merging**